tests/os.rs

```diff
@@ -0,0 +1,133 @@
+mod common;
+
+use blockifier::block_context::BlockContext;
+use blockifier::invoke_tx_args;
+use blockifier::test_utils::{create_calldata, NonceManager};
+use blockifier::transaction::test_utils;
+use blockifier::transaction::test_utils::max_fee;
+use rstest::rstest;
+use snos::config::STORED_BLOCK_HASH_BUFFER;
+use starknet_api::hash::StarkFelt;
+use starknet_api::stark_felt;
+use starknet_api::transaction::{Fee, TransactionVersion};
+
+use crate::common::block_context;
+use crate::common::state::{initial_state, InitialState};
+use crate::common::transaction_utils::execute_txs_and_run_os;
+
+#[rstest]
+fn simple_method_cairo0(block_context: BlockContext, initial_state: InitialState, max_fee: Fee) {
```

Comment:
Isn't there a better name than  `simple_method`? Feels pretty generic.

Reply:
Changed to: `return_result_cairo0_account`, `return_result_cairo1_account`.